package.json

```diff
@@ -60,7 +60,7 @@
     "yalc": "^1.0.0-pre.50"
   },
   "peerDependencies": {
-    "preact": "^10.4.0"
+    "preact": "^10.25.1"
```

Comment:
Setting minimum required version to 10.25.1 rather than 10.25.0, since it includes some important type fixes.